src/libutil/experimental-features.cc

```diff
@@ -214,6 +214,22 @@ constexpr std::array<ExperimentalFeatureDetails, 13> xpFeatureDetails = {{
                 derivations that are themselves derivations outputs.
         )",
     },
+    {
+        .tag = Xp::ReadOnlyLocalStore,
+        .name = "read-only-local-store",
+        .description = R"(
+            Allow the use of the `read-only` parameter in local store URIs.
```

Comment:
Would be great if you could also take on https://github.com/NixOS/nix/issues/8496 so we can link to parameters properly. That way the new feature will be way more useful.

Reply:
```suggestion
            Allow the use of the `read-only` parameter in local [store URLs](@docroot@/command-ref/new-cli/nix3-help-stores.md).
```

Not sure if the link is correct, but at least we have something to point people to. Implementing #8496 will break that link so it can be updated to point to the right thing.